accessibility-checker-engine/README-NPM.md

```diff
@@ -212,4 +212,4 @@ You can use the [karma-accessibility-checker](https://www.npmjs.com/package/karm
 
 If you think you've found a bug, have questions or suggestions, please report the bug in [GitHub Issues](https://github.com/IBMa/equal-access/issues).
 
-This software includes material copied from or derived from the open ACT-Rules Community. Copyright © 2022 W3C® (MIT, ERCIM, Keio, Beihang).
\ No newline at end of file
+This software includes material copied from or derived from the open ACT-Rules Community. Copyright © 2022 W3C® (MIT, ERCIM, Keio, Beijing ).
```

Comment:
```suggestion
This software includes material copied from or derived from the open ACT-Rules Community. Copyright © 2022 W3C® (MIT, ERCIM, Keio, Beihang).
```

Reply:
Beihang is name of University, while Beijing is name of city